src/main/tv/codely/scala_intro_examples/lesson_08_collections/WordAggregator.scala

```diff
@@ -4,6 +4,10 @@ final class WordAggregator {
   // @ToDo: Here you have your exercise!
   // @see tv.codely.scala_intro_examples.lesson_08_collections.WordAggregatorSpec
   def aggregateWords(text: String): Map[String, Int] = {
-    Map.empty
+    if (text.length == 0) Map.empty[String,Int]
```

Comment:
It isn't needed to specify the types of an empty `Map`. That is, you can replace `Map.empty[String, Int]` for `Map.empty`. This contract is already specified in the method signatures, so we'll continue exposing a `Map[String, Int]` despite not being explicitly indicating so in the `Map.empty` call 🙂